lib/lveditqmljs/src/codeqmlhandler.cpp

```diff
@@ -300,6 +301,20 @@ void CodeQmlHandler::assistCompletion(
             cursorChange.endEditBlock();
             cursorChange.movePosition(QTextCursor::Left, QTextCursor::MoveAnchor, 1);
             return;
+        } else if (insertion == '}'){
+            cursorChange = cursor;
+            if (cursor.position() > 5)
+            {
+                QString text = m_target->toPlainText();
```

Comment:
This will be quite inefficient for large documents, since it needs to convert all the lines into a string. We should use another `QTextCursor` just to select the text between. (`QTextCursor::setPosition(..)`, then `QTextCurosr::setPosition(...QTextCursor::KeepAnchor)` to keep the selection.

Reply:
Is a block-level solution acceptable? Finding a block via position, and then editing that in a similar way?